test/datadog.spec.ts

```diff
@@ -153,3 +153,43 @@ describe("validateProps", () => {
     expect(thrownError?.message).toEqual("When `extensionLayer` is set, `apiKey` or `apiKmsKey` must also be set.");
   });
 });
+
+describe("addCdkTag", () => {
+  it("adds the dd_cdk_construct tag to all lambda function", () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "sa-east-1",
+      },
+    });
+    const hello1 = new lambda.Function(stack, "HelloHandler1", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    const hello2 = new lambda.Function(stack, "HelloHandler2", {
+      runtime: lambda.Runtime.PYTHON_3_8,
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    addCdkTag([hello1, hello2]);
+    expect(stack).toHaveResourceLike("AWS::Lambda::Function", {
+      Runtime: "nodejs10.x",
+      Tags: [
+        {
+          Key: "dd_cdk_construct",
+          Value: "v[object Object]",
+        },
+      ],
+    });
+    expect(stack).toHaveResourceLike("AWS::Lambda::Function", {
+      Runtime: "python3.8",
+      Tags: [
+        {
+          Key: "dd_cdk_construct",
+          Value: "v[object Object]",
```

Comment:
CDK doesn't fill the `[object Object]` part with the number until its actually deployed

Reply:
Might make sense to leave that as a comment here in the test

Reply:
This is a mistake, check the new commit